src/transaction_builder.js

```diff
@@ -272,18 +272,36 @@ export class TransactionBuilder {
   }
 
   /**
-   * Builds a {@link FeeBumpTransaction}
-   * @param {Keypair} feeSource - The account paying for the transaction.
-   * @param {string} baseFee - The max fee willing to pay per operation in inner transaction (**in stroops**). Required.
-   * @param {Transaction} innerTx - The Transaction to be bumped by the fee bump transaction.
-   * @param {string} networkPassphrase - networkPassphrase of the target stellar network (e.g. "Public Global Stellar Network ; September 2015").
+   * Builds a {@link FeeBumpTransaction}, enabling you to resubmit an existing
+   * transaction with a higher fee.
+   *
+   * @param {Keypair}     feeSource         - account paying for the transaction
+   * @param {string}      baseFee           - max fee willing to pay per
+   *     operation in inner transaction (**in stroops**)
+   * @param {Transaction} innerTx           - {@link Transaction} to be bumped
+   *     by the fee bump transaction
+   * @param {string}      networkPassphrase - passphrase of the target Stellar
+   *     network (e.g. "Public Global Stellar Network ; September 2015")
+   * @param {string}      [muxedId]         - an (optional, stringified) ID to
+   *     indicate that the source account for the fee-bump is a muxed account.
+   *     The muxed account will be built from the `feeSource`'s public key and
+   *     this `muxedId`.
+   *
+   * @todo Alongside the next major version bump, this type signature can be
+   *       changed to be less awkward: accept a MuxedAccount as the `feeSource`
+   *       rather than a keypair and an optional ID.
+   *
+   * @note Your fee-bump amount should be 10x the original fee.
+   * @see  https://developers.stellar.org/docs/glossary/fee-bumps/#replace-by-fee
+   *
    * @returns {FeeBumpTransaction}
    */
   static buildFeeBumpTransaction(
     feeSource,
     baseFee,
     innerTx,
-    networkPassphrase
+    networkPassphrase,
+    muxedId
```

Comment:
Is it possible for us to make feeSource a union of `Keypair` or `string`, where `string` is an G/M address?

That should be backwards compatible.

That would be more consistent with other places in the SDK where we can provide a source account in string form too.

Reply:
Unfortunately, no: it has to be a `Keypair` because it signs the transaction.

Reply:
Woah. Building a fee bump signs it? That's not good. 

When I look through the `buildFeeBumpTransaction` I don't see the keypair being used to sign. The signatures are empty here:
https://github.com/stellar/js-stellar-base/blob/d92bb4a04d89b4def0b9e1a93204ab8558579913/src/transaction_builder.js#L358

Reply:
Uhm wow, yeah, you're right. I could've sworn.. In that case, I definitely support this idea! Will give it a shot now.

Reply:
Done in 0dd5b8d; let me know what you think!